vissl/utils/hydra_config.py

```diff
@@ -116,6 +121,7 @@ def convert_to_attrdict(cfg: DictConfig, cmdline_args: List[Any] = None):
 
     # assert the config and infer
     config = cfg.config
+    save_attrdict_to_disk(cfg)
```

Comment:
we should save this after the `assert_hydra_conf()` call in next line. The assert_hydra_conf automatically infers and adjusts a few values in the config files. :) sorry if I missed this detail in our task description.